lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -109,6 +109,22 @@
                                 options: fieldsData.country.options,
                                 persistChanges: true
                             })
+                        },
+                        {
+                            view: new AccountSettingsFieldViews.DropdownFieldView({
+                                model: userPreferencesModel,
+                                required: true,
```

Comment:
In the dropdown, there's a blank option initially when the user hasn't selected anything. Setting `required: true` gets rid of the blank option once the users selects a time zone.